### PR TITLE
Fix broken links

### PR DESCRIPTION
**Pull request by kurianbenoy**
# What does this PR do?

- Fix broken links in page: https://huggingface.co/docs/transformers/model_doc/marian

Fixes # (issue)

## Before submitting
- [x] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you read the [contributor guideline](https://github.com/huggingface/transformers/blob/master/CONTRIBUTING.md#start-contributing-pull-requests),
      Pull Request section?
- [ ] Was this discussed/approved via a Github issue or the [forum](https://discuss.huggingface.co/)? Please add a link
      to it if that's the case.
- [ ] Did you make sure to update the documentation with your changes? Here are the
      [documentation guidelines](https://github.com/huggingface/transformers/tree/master/docs), and
      [here are tips on formatting docstrings](https://github.com/huggingface/transformers/tree/master/docs#writing-source-documentation).
- [ ] Did you write any new necessary tests?

## Who can review?

Anyone in the community is free to review the PR once the tests have passed. Feel free to tag
members/contributors who may be interested in your PR.

### CODE REVIEW

File: docs/source/model_doc/marian.mdx [hunk 1]
```diff
@@ -48,8 +48,7 @@ Translations should be similar, but not identical to output in the test set link
 
 - Since Marian models are smaller than many other translation models available in the library, they can be useful for
   fine-tuning experiments and integration tests.
-- [Fine-tune on GPU](https://github.com/huggingface/transformers/blob/master/examples/research_projects/seq2seq-distillation/train_distil_marian_enro_teacher.sh)
-- [Fine-tune on GPU with pytorch-lightning](https://github.com/huggingface/transformers/blob/master/examples/research_projects/seq2seq-distillation/train_distil_marian_no_teacher.sh)
+- [Fine-tune on GPU](thttps://github.com/huggingface/transformers/blob/master/examples/legacy/seq2seq/train_distil_marian_enro.sh)
```
Comment (LysandreJik) [inline docs/source/model_doc/marian.mdx:51]:
```suggestion
- [Fine-tune on GPU](https://github.com/huggingface/transformers/blob/master/examples/legacy/seq2seq/train_distil_marian_enro.sh)
```

**Review by LysandreJik — COMMENTED** [review]
Cool!

**Comment by kurianbenoy** [comment]
Have updated the documentation. Thanks for catch @LysandreJik

**Review by LysandreJik — APPROVED** [review]
LGTM!

**Pull request merged**